apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -268,7 +268,14 @@ internal V1.Type ToProtobufType()
         /// </summary>
         /// <param name="size"></param>
         /// <returns></returns>
-        public SpannerDbType WithSize(int size) => new SpannerDbType(TypeCode, size);
+        public SpannerDbType WithSize(int size)
```

Comment:
Please add documentation.

Reply:
this is actually pretty important because size is ambiguous for string.  for spanner, its # of characters.

Reply:
What does "characters" mean here? Unicode code points, UTF-16 code units, UTF-8 bytes? (Often the last is used for databases if UTF-8 is raw storage :)

Reply:
'characters' is the term used in public spanner documentation and i think it makes sense to be consistent.  While "code point" is probably the more exact term that covers formatting, it seems a bit unnecessary and overly technical.  I've never heard anyone refer to unicode 'characters' as 'code points' in vernacular. 

Specifying UTF-8 at all might be unnecessarily exposing an implementation detail since the encoding is enforced server side independent of the transport.

Reply:
Right, if it's definitely Unicode characters then the encoding is irrelevant. (I assume it's not going to perform any kind of normalization.)

Whether we need to mention that the size in characters isn't necessarily the value returned by `string.Length` is a judgement call...